crates/pathfinder/src/rpc/serde.rs

```diff
@@ -289,6 +289,13 @@ serde_with::serde_conv!(
     |s: &str| bytes_from_hex_str::<{ H256::len_bytes() }>(s).map(|b| TransactionVersion(H256::from(b)))
 );
 
+serde_with::serde_conv!(
+    pub H256AsHexStr,
+    web3::types::H256,
+    |u: &H256| bytes_to_hex_str(u.as_bytes()),
+    |s: &str| bytes_from_hex_str::<32>(s).map(H256::from)
+);
+
```

Comment:
I never tested using just the default deserialization on U256, perhaps it would had worked. Does anyone know? It'd be preferable to send the values out variable length, but python side communicates them fixed length.

Reply:
As far as I remember, `U256` only mapped to / from decimal numbers. I recall having issues with the L1 logs when using the wrong type.

Not 100% certain of this though.